settings.gradle

```diff
@@ -26,3 +26,5 @@ include "eventeria-timer-spring-integration"
 include "eventeria-messaging-timer-spring-redis"
 include "spring-boot-eventeria"
 include "eventeria-fake-spring-cloud-stream-binder-kafka"
+
```

Comment:
is it an intentional newline?

Reply:
Yes. I added this line to separate publishing modules and guide proejcts